data/test_data.json

```diff
@@ -79,15 +87,16 @@
       "longitude": "73.8648",
       "realtor_id": 1,
       "city_id": 4,
-      "amenities":["test_amenities1"],
+      "amenities": [
+        "test_amenities1"
+      ],
```

Comment:
Why do we need these amenities here?

Reply:
It does not look that they can be used in tests

Reply:
I didn't add it, so I don't even know whether to delete it!